.github/workflows/ci.yml

```
@@ -644,3 +673,121 @@ jobs:
         run: python3 ./qa/zcash/full_test_suite.py sec-hard
         env:
           HOST: ${{ matrix.host }}
+
+  rpc-depends:
+    name: RPC set up tier ${{ matrix.tier }} platform ${{ matrix.platform }}
+    needs: setup
+    runs-on: ${{ matrix.build_os }}
```

This job only runs for a few seconds, so it should use `test_os` to avoid costing an entire minute of the build runners:
```suggestion
    runs-on: ${{ matrix.test_os }}
```